code/modules/mob/living/carbon/human/species_types.dm

```diff
@@ -549,6 +554,7 @@ var/global/list/synth_flesh_disguises = list()
 	SA_para_min = 0
 	SA_sleep_min = 0
 	dangerous_existence = 1
```

Comment:
@Incoming5643 actually, I'm going to keep this, as I've seen the synths not get their disguises built properly, leaving them to look really weird.